packages/angular/package.json

```
@@ -20,7 +20,7 @@
         "prepublishOnly": "cp ./package.json ./dist && cp ../../README.md ./dist"
     },
     "peerDependencies": {
-        "@angular/common": ">=12",
+        "@angular/common": ">=13",
```

Today while updating `eslint-config` I had to change this, as it wasn't working. [The change](https://github.com/mobilejazz/eslint-config/commit/a170c6187395af0b4772ed2cfe63c78488990bd5) follows the suggestion from here: [Make your NPM package support multiple versions of peer dependency](https://stackoverflow.com/questions/47309598/make-your-npm-package-support-multiple-versions-of-peer-dependency).

```suggestion
        "@angular/common": "13 - 15",
```

We could also do what some other libraries do: `^13.0.0 || ^14.0.0 || ^15.0.0`. But it's extra verbose and you need to remember what `^` does (allow different minor/patch).

I've created a task earlier, so now that you're with this I'm assigning it to you: https://app.asana.com/0/1109863238977521/1203409192207937/f

---

Actually, I'm not sure about this… I've just double checked in: https://semver.npmjs.com And `>=13` should be OK. I'm not sure what was the issue I was having in `esling-config`.